### PR TITLE
Update documentation for agent.serverCertificate

### PR DESCRIPTION
**Pull request by APErebus**
Also updated main documentation link.

Will be forward merged to `main`.

I don't think this needs a release as it's just a documentation update

### CODE REVIEW

File: charts/kubernetes-agent/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ A Helm chart for the Octopus Kubernetes Agent
 | agent.securityContext | object | `{}` | The security context to apply to the agent pod. runAsGroup and fsGroup should be blank or set to `0` |
 | agent.serverApiKey | string | `""` | An Octopus Server API key used to authenticate with the target Octopus Server |
 | agent.serverApiKeySecretName | string | `""` | The name of an existing Secret that contains a base64-encoded Octopus Server API Key.  Value must be set in `data.api-key` in secret. |
-| agent.serverCertificate | string | `""` | The base64-encoded public x509 certificate used by the target Octopus Server. Must be in the PEM/CER format. |
+| agent.serverCertificate | string | `""` | The base64-encoded public key of the self-signed x509 certificate or root CA certificate used by the target Octopus Server. Must be in the PEM/CER format. See [documentation](https://octopus.com/docs/kubernetes/targets/kubernetes-agent#trusting-custominternal-octopus-server-certificates) for more information. |
```
Comment (kevjt) [inline charts/kubernetes-agent/README.md:45]: Have you tested that this link works? I'm assuming the `/` in `custom/internal` is ignored.

**Review by kevjt — APPROVED** [review]
lgtm

File: charts/kubernetes-agent/README.md (hunk 1 above)
Comment (APErebus) [reply charts/kubernetes-agent/README.md:45]: I haven't. I shall check

Comment (APErebus) [reply charts/kubernetes-agent/README.md:45]: Yep, it works :)

**Pull request merged**